daliuge-engine/dlg/data/drops/memory.py

```diff
@@ -33,6 +33,9 @@
 from dlg.common.reproducibility.reproducibility import common_hash
 from dlg.data.drops.data_base import DataDROP, logger
 from dlg.data.io import SharedMemoryIO, MemoryIO
+from dlg.utils import object_tracking
+
+track_current_drop = object_tracking("drop")
```

Comment:
We don't want to create a new object tracker - this has caused issues for us before: https://github.com/ICRAR/daliuge/pull/276

What happens is that the `_DlgLogger` class is specifically importing the `dlg.drop.track_current_drop` that is set up. This will create a new object tracker that is not processed by the logger.